tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -144,90 +119,80 @@ def test_callf_stack_size_1024_at_push(
                 ),
             ],
         ),
-        container_post=Account(storage={slot_code_worked: value_code_worked}),
-    )
-
-
-def test_callf_stack_overflow(
-    eof_state_test: EOFStateTestFiller,
-):
-    """Test stack overflowing 1024 items in called function"""
-    eof_state_test(
-        data=Container(
+        Container(
+            name="nested_with_inputs_at_push0",
             sections=[
                 Section.Code(
-                    code=Op.PUSH0 * 1023
+                    code=Op.PUSH0 * 1022
```

Comment:
This diff is because there was one test for `stack_size_max` after `stack_overflow`.